python/hsfs/core/feature_monitoring_result_engine.py

```diff
@@ -66,23 +66,23 @@ def save_feature_monitoring_result(
         self,
         config_id: int,
         execution_id: int,
-        detection_stats_id: int,
```

Comment:
Don't we want to keep those if we don't fetch the results with expand=statistics?

Reply:
You are right

Reply:
Ja sorry, thought wrong here